start.py

```diff
@@ -20,9 +21,37 @@
 from clusterdock.utils import wait_for_condition
 
 DEFAULT_NAMESPACE = 'clusterdock'
+ZOOKEEPER_PORT = 2181
+BROKER_PORT = 9092
 
 logger = logging.getLogger('clusterdock.{}'.format(__name__))
 
+def success(time):
+    logger.info('Conditions satisfied after %s seconds.', time)
+
+
+def failure(timeout):
+    raise TimeoutError('Timed out after {} seconds waiting.'.format(timeout))
+
+
+# Validate that Zookeeper is up and running by connecting using shell
+def validate_zookeeper(node):
+    return node.execute('/kafka/bin/zookeeper-shell.sh localhost:2181 ls /', quiet=True).exit_code == 0
```

Comment:
Instead of setting quiet to True or false, it's better to set a variable in function with `quiet = not args.verbose` so that this can be toggled by people who want to run `clusterdock -v start ...` Do same throughout anywhere `quiet=True` is set.